src/docs/content/builders/dialog.md

```diff
@@ -65,7 +65,7 @@ an alert dialog would be appropriate.
 By default, we set the `role` attribute to `dialog`. If you want it to be considered an alert
 dialog, you can set the `role` builder prop to `alertdialog`.
 
-```ts {2}
+```ts {4}
 const {
 	/* ... */
 } = createDialog({
```

Comment:
Instead, maybe we should just remove all this extra space/line-breaks around the comment? Same for all of them

Reply:
okay then, removing the spaces